app/src/main/java/com/example/firstandroid/ViewModel/UpdateBathroom/UpdateBathroomViewModel.kt

```diff
@@ -14,40 +14,40 @@ import androidx.compose.runtime.rememberCoroutineScope
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.unit.dp
-import com.example.firstandroid.ViewModel.NewBathroom.NewBathroomViewModel
 import kotlinx.coroutines.launch
+import java.math.BigDecimal
 
```

Comment:
Why is the file name say ViewModel but then the contents are a Composable?

I think organization reset is needed for the project.